Error: prefix already exists: /home/ubuntu/miniconda/envs/_pipbuild_
Collecting python-ceilometerclient==1.3.0
  Could not find a version that satisfies the requirement python-ceilometerclient==1.3.0 (from versions: 1.0.0, 1.0.1, 1.0.2, 1.0.3, 1.0.4, 1.0.5, 1.0.6, 1.0.7, 1.0.8, 1.0.9, 1.0.10, 1.0.11, 1.0.12, 1.0.13, 1.0.14, 1.1.0, 1.2.0)
No matching distribution found for python-ceilometerclient==1.3.0
Package search: {'_pypi_ordering': False, 'version': '1.3.0', 'name': 'python-ceilometerclient', 'summary': 'OpenStack Telemetry API Client Library'}
Creating standard recipe for python-ceilometerclient-1.3.0
Using url https://pypi.python.org/packages/source/p/python-ceilometerclient/python-ceilometerclient-1.3.0.tar.gz (91 KB) for python-ceilometerclient.
Downloading python-ceilometerclient
Unpacking python-ceilometerclient...
done
working in /tmp/tmpNaZIxbconda_skeleton_python-ceilometerclient-1.3.0.tar.gz
Fetching package metadata: ......

WARNING: conda-build appears to be out of date. You have version 1.13.0 but the
latest version is 1.14.0. Run

conda update -n root conda-build

to get the latest version.

Solving package specifications: .patching file core.py
Hunk #1 succeeded at 168 with fuzz 2 (offset 2 lines).
Traceback (most recent call last):
  File "setup.py", line 18, in <module>
    import setuptools
ImportError: No module named setuptools

Applying patch: '/tmp/tmpNaZIxbconda_skeleton_python-ceilometerclient-1.3.0.tar.gz/pypi-distutils.patch'
$PYTHONPATH = /tmp/tmpNaZIxbconda_skeleton_python-ceilometerclient-1.3.0.tar.gz/python-ceilometerclient-1.3.0
Error: command failed: /home/ubuntu/miniconda/envs/_build/bin/python setup.py install

conda create -n _pipbuild_ --yes python pip
/home/ubuntu/miniconda/envs/_pipbuild_/bin/pip install python-ceilometerclient==1.3.0
Error: Could not pip install python-ceilometerclient==1.3.0
